app/database/migrations/2015_11_08_222305_add_ldap_fields_to_settings.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddLdapFieldsToSettings extends Migration {
+
+	/**
+	 * Run the migrations.
+	 * 
+	 * @return void
+	 */
+	public function up()
+	{
+		Schema::table('settings', function(Blueprint $table)
+		{
+			$table->string('ldap_enabled')->nullable()->default(NULL);
+			$table->string('ldap_server')->nullable()->default(NULL);
+			$table->string('ldap_uname')->nullable()->default(NULL);
+			$table->longText('ldap_pword')->nullable()->default('NULL');
```

Comment:
You have `->default('NULL')` instead of `->default(NULL)`, so you're giving it the literal value of NULL, versus a null value. I can fix this though.